From: Matthias Goergens <matthias.goergens@citrix.com>
Subject: [PATCH] Cheated on required cabal version.

Building works with the lower version that we have.  The newfangled
sections are ignored, and we don't need them.

TODO: Remove this patch, once we switch to Cabal-version: >= 1.10


diff --git a/semigroups.cabal b/semigroups.cabal
index f42d419..1017de5 100644
--- a/semigroups.cabal
+++ b/semigroups.cabal
@@ -2,7 +2,7 @@ name:          semigroups
 category:      Algebra, Data, Data Structures, Math
 version:       0.8.4
 license:       BSD3
-cabal-version: >= 1.10
+cabal-version: >= 1.6
 license-file:  LICENSE
 author:        Edward A. Kmett
 maintainer:    Edward A. Kmett <ekmett@gmail.com>
